mesa/visualization/components/matplotlib.py

```diff
@@ -52,16 +52,20 @@ def SpaceMatplotlib(
     if space is None:
         space = getattr(model, "space", None)
 
-    if isinstance(space, mesa.space._Grid):
-        _draw_grid(space, space_ax, agent_portrayal, propertylayer_portrayal, model)
```

Comment:
Hi @quaquel, I understand that this is already merged. May I ask whether it is intentional to remove the call to `_draw_grid()` here? Thanks!

Reply:
I guess I made a mistake. I'll put in a PR to fix this asap.

Thanks for the post-merge review!

Reply:
and fixed via #2398

Reply:
Great! Thanks!